lib/LaTeXML/resources/XSLT/LaTeXML-common.xsl

```diff
@@ -151,7 +151,7 @@
   -->
   <func:function name="f:url">
     <xsl:param name="url"/>
-    <func:result><xsl:value-of select="$url"/></func:result>
+    <func:result><xsl:value-of select="f:subst($url,'&amp;','%26')"/></func:result>
```

Comment:
I think this breaks query strings: `&` should not be URL-encoded if it is separating different arguments after `?`. As far as I know, `&amp;` is fine inside an `@href` and will be correctly interpreted as a normal `&`. As long as it doesn't get *doubly* serialised, i.e. it doesn't become `&amp;amp;`. But that would be a different bug.

Reply:
I think I managed to confuse myself here by testing the URLs in vacuum, rather than by clicking on `<a>` elements in HTML documents, where it would have become visible that the `&amp;` works just fine.

Thank you for catching that @xworld21 , I will undo the change.